radix-engine/src/blueprints/consensus_manager/package.rs

```diff
@@ -16,18 +16,20 @@ use resources_tracker_macro::trace_resources;
 
 use super::*;
 
-pub struct EpochManagerNativePackage;
+pub struct ConsensusManagerNativePackage;
 
-impl EpochManagerNativePackage {
+impl ConsensusManagerNativePackage {
     pub fn schema() -> PackageSchema {
         let mut aggregator = TypeAggregator::<ScryptoCustomTypeKind>::new();
 
         let mut fields = Vec::new();
-        fields.push(aggregator.add_child_type_and_descendents::<EpochManagerConfigSubstate>());
-        fields.push(aggregator.add_child_type_and_descendents::<EpochManagerSubstate>());
+        fields.push(aggregator.add_child_type_and_descendents::<ConsensusManagerConfigSubstate>());
+        fields.push(aggregator.add_child_type_and_descendents::<ConsensusManagerSubstate>());
         fields.push(aggregator.add_child_type_and_descendents::<CurrentValidatorSetSubstate>());
         fields
             .push(aggregator.add_child_type_and_descendents::<CurrentProposalStatisticSubstate>());
+        fields.push(aggregator.add_child_type_and_descendents::<i64>());
```

Comment:
Would you be up for giving these substates explicit names? Eg `ProposerTimestampRoundedToPreviousMinute` / `LatestProposerTimestamp` or similar? :)

They can either be type aliases (with `type`) or explicit singleton structs, possibly with `#[sbor(transparent)]` if we want to just store them as `i64` for performance.